bool sightedALine
int32 durationSinceSighting
int32 lapNum
bool sightedFinish

